code/modules/antagonists/heretic/knowledge/blade_lore.dm

```diff
@@ -372,13 +413,20 @@
 
 	// Save the force as our last weapon force
 	last_weapon_force = blade.force
-	// Subtract the decrement
-	blade.force -= offand_force_decrement
+	// Subtract the decrement, but only if the target is living
+	if(isliving(target))
+		blade.force -= offand_force_decrement
 	// Perform the offhand attack
 	blade.melee_attack_chain(source, target)
 	// Restore the force.
 	blade.force = last_weapon_force
 
+///Modifies our blade demolition modifier so we can take down doors with it
+/datum/heretic_knowledge/blade_upgrade/blade/proc/on_blade_equipped(mob/user, obj/item/equipped, slot)
+	SIGNAL_HANDLER
+	if(istype(equipped, /obj/item/melee/sickly_blade/dark))
+		equipped.demolition_mod = 1.5
```

Comment:
You don't have to change it in this pr but I would love for us to have a framework for "knowledge that changes your blades in some way", this is kind of jank (but acceptable)